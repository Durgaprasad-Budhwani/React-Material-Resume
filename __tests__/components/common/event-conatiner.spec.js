/**
 * Created by Durgaprasad Budhwani on 10/6/2017.
 */
/* global expect,it */
import React from 'react';
import EventContainer from '../../../src/components/common/event-container';
import renderer from 'react-test-renderer';

it('renders event container for left', () => {
    const tree = renderer.create(<EventContainer alignment="LEFT" />).toJSON();
    expect(tree).toMatchSnapshot();
});

it('renders event container for right', () => {
    const tree = renderer.create(<EventContainer alignment="RIGHT" />).toJSON();
    expect(tree).toMatchSnapshot();
});
