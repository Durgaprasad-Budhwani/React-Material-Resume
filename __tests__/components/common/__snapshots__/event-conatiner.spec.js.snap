// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders event container for left 1`] = `
<div
  className="EventContainer-box-5 EventContainer-boxLeft-6"
>
  <div>
    <div
      className="MuiPaper-root-14 MuiPaper-shadow2-18 MuiPaper-rounded-15 EventContainer-paper-1 EventContainer-paperLeft-2"
    >
      <span
        className="EventContainer-arrow-9 EventContainer-arrowRight-10"
      />
    </div>
  </div>
  <span
    className="EventContainer-dot-8 EventContainer-dotRight-12"
  />
</div>
`;

exports[`renders event container for right 1`] = `
<div
  className="EventContainer-box-5 EventContainer-boxRight-7"
>
  <div>
    <div
      className="MuiPaper-root-14 MuiPaper-shadow2-18 MuiPaper-rounded-15 EventContainer-paper-1 EventContainer-paperRight-3"
    >
      <span
        className="EventContainer-arrow-9 EventContainer-arrowLeft-11"
      />
    </div>
  </div>
  <span
    className="EventContainer-dot-8 EventContainer-dotLeft-13"
  />
</div>
`;
